dask/highlevelgraph.py

```diff
@@ -870,6 +870,51 @@ def merge(cls, *graphs):
         return cls(layers, dependencies)
 
     def visualize(self, filename="dask-hlg.svg", format=None, **kwargs):
+        """
+        Visualize several dask high level graphs at once.
```

Comment:
Just one high level graph!

Reply:
Yeah, I asked about this wording too. Turns out @freyam copied it from the low level graph visualize function (which is what I suggested he look at to base this off). I don't think it's true for either situation - am I wrong?

If that's the case, we'll want to update both docstrings to say something like this instead:
* `"Visualize a dask high level graph."`
* `"Visualize a dask low level task graph."`

Reply:
Yes, +1 here. I am not sure if we can pass several dictionaries into visualize. I tried it myself and I am getting errors.

I have updated the doc to the lines Genevieve sent.